src/style.css

```diff
@@ -0,0 +1,61 @@
+.body {
+  margin: 0;
+}
+
+.container {
+  width: 96px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 > .stars_star {
```

Comment:
You have a lot of repeated lines of code, please fix it all according to the example. You can add this rule as default for stars_star `background-image: url(./images/star.svg);`. And below only assign star-active
![image](https://user-images.githubusercontent.com/54936044/230994644-8c70a6e2-8253-423c-a91d-ec217493218e.png)